### PR TITLE
Add system-logs.md's zh translation

### PR DESCRIPTION
**Pull request by xieyanker**

### CODE REVIEW

File: content/zh/docs/concepts/cluster-administration/system-logs.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+-->
+
+`-v` 标志控制日志的详细程度。增大该值将增加记录的事件数，减小该值将减少记录的事件数。
+随着该值的逐渐增加记录的事件严重程度会越来越低。当级别设置为 0 时仅记录关键事件。
+
+<!--
+### Log location
+-->
+
+### 记录位置
```
Comment (tengqm) [inline content/zh/docs/concepts/cluster-administration/system-logs.md:186]: 日志位置

File: content/zh/docs/concepts/cluster-administration/system-logs.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
+---
+reviewers:
+- dims
+- 44past4
```
Comment (tengqm) [inline content/zh/docs/concepts/cluster-administration/system-logs.md:4]: remove these lines

**Comment by xieyanker** [comment]
@tengqm Thanks for your correction, I have updated them, please review!

**Comment by tengqm** [comment]
@xieyanker Thanks for the contribution. Looks like  this is a dup of #24025?

**Comment by xieyanker** [comment]
@tengqm  OK, I didn't notice that, you can close this PR.

**Pull request closed without merging**